### PR TITLE
[IMP] web: speed up pager total counter for large DB

### PR DESCRIPTION
**Pull request by aab-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/views/relational_model.js [hunk 1]
```diff
@@ -1789,6 +1791,20 @@ export class DynamicRecordList extends DynamicList {
...
 
+    /**
+     * Performs a search_count with the current domain to set the count. This is
+     * useful as web_search_read limits the count for performance reasons, so it
+     * might sometimes be less than the real number of records matching the domain.
+     *
+     * @returns {number}
+     */
+    async fetchCount() {
+        this.count = await this.model.orm.searchCount(this.resModel, this.domain);
```
Comment (ged-odoo) [inline addons/web/static/src/views/relational_model.js:1802]: maybe should have a dropprevious?

**Review by ged-odoo — APPROVED** [review]

**Comment by ged-odoo** [comment]
robodoo r+

**Comment by ged-odoo** [comment]
robodoo r-

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**